src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -697,7 +697,7 @@ void AtlasEngine::_flushBufferLine()
                 codepoint = til::combine_surrogates(codepoint, beg[i++]);
             }
 
-            const auto c = BuiltinGlyphs::IsCustomGlyph(codepoint) || BuiltinGlyphs::IsSoftFontChar(codepoint);
+            const auto c = BuiltinGlyphs::IsBuiltinGlyph(codepoint) || BuiltinGlyphs::IsSoftFontChar(codepoint);

```

Comment:
(it still calls both!)

Reply:
Yes, it needs to! `IsBuiltinGlyph` is only for the builtin box glyphs.
`BackendD3D` then checks again if it's a `IsSoftFontChar` and otherwise it assumes it's a `IsBuiltinGlyph`. Only in case of the latter it'll then call `DrawBuiltinGlyph`.